pkgs/tools/networking/slackcat/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "slackcat";
+  version = "1.6";
+
+  src = fetchFromGitHub {
+    owner = "bcicen";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "1ha9b2dffpxqs159ldqdh1aibn3316xab19vdn2yjgk8r508wm31";
+  };
```

Comment:
I noticed they have a go.mod https://github.com/bcicen/slackcat/blob/0b70f6e0ab9c41e4782bfbe74e934ddb49d520b8/go.mod, do you mind trying to do a `buildGoModule` instead? package is deprecated